src/main/scala/fake/FakeMessage.scala

```diff
@@ -0,0 +1,16 @@
+package fake
+
+import java.util.Date
+
+/**
+ * Created by cwheeler on 5/18/15.
+ */
+case class FakeMessage(messageId: String, messageContent: String, timestamp: Date) extends Serializable
```

Comment:
Cleaned up your 'class' and made it a scala case class with the appropriate 'apply' function